main.go

```diff
@@ -31,8 +31,11 @@ func main() {
 				ShowExecEnv:           c.Bool("show-exec-env"),
 				OutputFormat:          c.String("output"),
 				PerfBufferSize:        c.Int("perf-buffer-size"),
+				BlobPerfBufferSize:    c.Int("blob-perf-buffer-size"),
 				OutputPath:            c.String("output-path"),
 				FilterFileWrite:       c.StringSlice("filter-file-write"),
+				ExtractDynamicCode:    c.Bool("extract-dynamic-code"),
```

Comment:
I thought this should be an option for `capture-files`, no? today we have `write/exec` we an add `mem` capture type.

Reply:
Yes, I wanted to do this, but then I thought that:
1. This is not a file, so capture-files not describes it well
2. The description of capture-files help got really messy when combining both:
"capture files that were written or that were executed. captured files will apear in the 'output-path' directory. possible values: 'write'/'exec'/'all'. use this flag multiple times to choose multiple file types"
+
"attempt to extract dynamically loaded code (e.g. using packers)"

Reply:
Maybe we can call it just "capture"? WDYT?
How would you describe the comment then?

Reply:
let's change to capture then

Reply:
> capture artifacts that were written, executed or found to be suspicious. captured files will appear in the 'output-path' directory. possible values: 'write'/'exec'/'mem'/'all'. use this flag multiple times to choose multiple file types"